template.yaml

```diff
@@ -270,44 +270,49 @@ Resources:
               - sts:TagSession
       Path: /
       Policies:
-        - PolicyName: CuratorAccess
+        - PolicyName: ReadAccess
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - dynamodb:GetItem
+                  - dynamodb:Query
+                  - dynamodb:BatchGetItem
+                  - dynamodb:DescribeTable
+                  - sts:AssumeRole
+                  - sts:TagSession
+                Resource:
+                  - !Join [ "",[ "arn:aws:dynamodb",":",!Ref AWS::Region,":",!Ref AWS::AccountId,":","table/",!Ref DoiRequestsTableName,"/index/",!Ref DoiRequestsIndexName ] ]
+                Condition:
+                  "ForAllValues:StringEquals":
```

Comment:
Condition for allowing access only to the index entries with partition key equal to the principal identifier (doiRequest index)